EIPS/eip-4488.md

```diff
@@ -0,0 +1,73 @@
+---
+eip: 4488
+title: Transaction calldata gas cost reduction with total calldata limit
+description: Greatly decreases the gas cost of transaction calldata and simultaneously caps total transaction calldata in a block
+author: Vitalik Buterin (@vbuterin), Ansgar Dietrichs (@adietrichs)
+discussions-to: https://ethereum-magicians.org/t/eip-4488-transaction-calldata-gas-cost-reduction-with-total-calldata-limit/7555
+type: Standards Track
+category: Core
+status: Draft
+created: 2021-11-23
+---
+
+## Abstract
+
+Decrease transaction calldata gas cost, and add a limit of how much total transaction calldata can be in a block.
+
+## Motivation
+
+Rollups are in the short and medium term, and possibly the long term, the only trustless scaling solution for Ethereum. Transaction fees on L1 have been very high for months and there is greater urgency in doing anything required to help facilitate an ecosystem-wide move to rollups. Rollups are significantly reducing fees for many Ethereum users: [l2fees.info](https://l2fees.info/) frequently shows Optimism and Arbitrum providing fees that are ~3-8x lower than the Ethereum base layer itself, and ZK rollups, which have better data compression and can avoid including signatures, have fees ~40-100x lower than the base layer.
```

Comment:
```suggestion
Rollups are in the short and medium term, and possibly the long term, the only trustless scaling solution for Ethereum. Transaction fees on L1 have been very high for months and there is greater urgency in doing anything required to help facilitate an ecosystem-wide move to rollups. Rollups are significantly reducing fees for many Ethereum users: Optimism and Arbitrum frequently provide fees that are ~3-8x lower than the Ethereum base layer itself, and ZK rollups, which have better data compression and can avoid including signatures, have fees ~40-100x lower than the base layer.
```
This link would likely be useless to a future reader.  It may be worthwhile to bring up in the discussion thread but it shouldn't be part of the EIP itself.